src/rust/src/address_ffi.rs

```diff
@@ -109,7 +110,7 @@ impl UnifiedAddressHelper {
                     (p2pkh_cb.unwrap())(ua_obj, data.as_ptr())
```

Comment:
It's correct that there are no error conditions for P2SH and P2PKH, because they are just 20-byte hashes and have no validity requirements.